isic_kredit/ViewControllers/CreditViewController.swift

```diff
@@ -0,0 +1,255 @@
+                                                ]
+                                                
+                                                
+                                                Alamofire.request(linkAction, method: .post, parameters: parameters1) .responseString { responseFinalSite in
+                                                    
+                                                    if let responseFinalSite = responseFinalSite.result.value{
+                                                        self.setMoneyValue(siteResultValue: responseFinalSite)
+                                                        self.hudSuccess(text: NSLocalizedString("Loaded", comment: ""))
+                                                    } else {
+                                                        self.hudError(text: NSLocalizedString("Fail", comment: ""))
+                                                        return
+                                                    }
+                                            
+                                                }
+                                                
+                                            } catch{
+                                                self.hudError(text: NSLocalizedString("Fail", comment: ""))
+                                                return
+                                            }
+                                            
+                                        }
+                                    }
+                                    
+                                }
+                            }
+                            
+                        }
+                    } else {
+                        self.hudError(text: NSLocalizedString("Fail", comment: ""))
+                        return
+                    }
+                    
+                  //User is logged in
+                } else {
+                    self.setMoneyValue(siteResultValue: (agataResponse.result.value)!)
+                    self.hudSuccess(text: NSLocalizedString("Loaded", comment: ""))
+                }
+                //There is something wrong with the site
+            } else {
+                self.hudError(text: NSLocalizedString("Fail", comment: ""))
+                return
+                
+            }
+            
+        }
+        
+    }
+    
+    /*
+     Gets money from the site and sets it to moneyLabel.
+     */
+    func setMoneyValue(siteResultValue: String){
+        do{
+            let agataWebSite:Document = try SwiftSoup.parse(siteResultValue)
+            let tbody: Element = try agataWebSite.select("tbody").first()!
+            let td: Element = try tbody.select("td").array()[4]
+            let arrayOfMoneyStrings = td.ownText().split(separator: ",")
+            self.moneyLabel.text = "\(arrayOfMoneyStrings[0]) \(NSLocalizedString("CZK", comment: ""))"
```

Comment:
Tady to bude crashovat, nemáš nikde check, že v tom poli něco je. A to si piš, že se stane mnohokrát, že tam nebude nic. Parsuješ HTML bez jakýkoliv dokumentace a příslibu, že tam něco bude nebo nebude, to musíš všechno ošetřit.